src/pages/result/components/filter-button/filter-button.component.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { ROUTES } from '../../../../shared/constants/routes.constants';
+
+import './filter-button.styles.scss';
+
+export const FilterButton: React.FC = () => {
+  return (
+    <div className="filter-button-container">
+      <Link to={ROUTES.search} className="filter-button">
```

Comment:
` <div className="filter-button-container">
      <Link to={ROUTES.search} className="filter-button">`
Please change to,  <Link to={ROUTES.filter} className="filter-button">
we changed the logic and now we have a separate page for the filter.

Reply:
I'll refactor it as soon as can! )